tree/master/packages/extension/public/locales/pl/translation.json

```
@@ -0,0 +1,162 @@
+{
+	  "Copied": "Skopiowane",
```

```suggestion
	  "Copied": "Skopiowano",
```
wydaje mi się bardziej odpowiednie jako reakcja na akcję użytkownika.